compiler/rustc_codegen_cranelift/src/base.rs

```diff
@@ -392,13 +392,20 @@ fn codegen_fn_content(fx: &mut FunctionCx<'_, '_, '_>) {
             TerminatorKind::Call {
                 func,
                 args,
-                destination,
+                dest_place,
+                dest_block,
                 fn_span,
                 cleanup: _,
                 from_hir_call: _,
             } => {
                 fx.tcx.sess.time("codegen call", || {
-                    crate::abi::codegen_terminator_call(fx, *fn_span, func, args, *destination)
+                    crate::abi::codegen_terminator_call(
+                        fx,
+                        *fn_span,
+                        func,
+                        args,
+                        dest_block.map(|x| (*dest_place, x)),
```

Comment:
Could you update `codegen_terminator_call` to take two separate arguements too?

Reply:
I updated that and propagated the change through to a bunch more functions whose signature benefited from this change. I don't think I did anything dangerous, but also I'm not really familiar with this code. Let me know if this change was more than you wanted, or if this should be adjusted in any other way